pandas-stubs/core/frame.pyi

```diff
@@ -89,7 +89,10 @@ class _iLocIndexerFrame(_iLocIndexer):
     @overload
     def __getitem__(
         self,
-        idx: IndexType | MaskType | tuple[IndexType | MaskType, IndexType | MaskType] | tuple[slice],
+        idx: IndexType
+        | MaskType
+        | tuple[IndexType | MaskType, IndexType | MaskType]
+        | tuple[slice],
```

Comment:
Sorry - I should have picked up on this before.  Use 
```python
tuple[IndexType | MaskType, IndexType | MaskType | slice]
```

Reply:
That change fails the test I made
```
Type "tuple[slice]" cannot be assigned to type "IndexType | MaskType | tuple[IndexType | MaskType, IndexType | MaskType]"
```

Reply:
My mistake.  I missed the comma